pkg/controller/import-controller.go

```diff
@@ -229,9 +229,8 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 	}
 
 	anno[AnnImportPod] = string(pod.Name)
-	if !scratchExitCode {
-		r.updatePvcPhaseFromPod(anno, pod)
-	}
+	// Even if scratch space is needed, the pod state will still remain running, until the new pod is started.
+	anno[AnnPodPhase] = string(pod.Status.Phase)
```

Comment:
I think that it is good to have an annotation that accurately reflects the pod phase.  But now there is no way for the user to know if there was a problem.  Perhaps they entered the url wrong?  This hit me after looking at the changes made to the unit tests.  Imports will never "fail" now.  May have to think about this a little more.

Reply:
Remember we emit events when we have a failure, and that event is noted in the PVC. And you are correct, if we are fully eventually consistent, then we will never 'fail' the import, we will keep retrying forever.

Reply:
Do the tests validate that the events are generated on failure?  If so then I ay be fine with this?

Reply:
https://github.com/kubevirt/containerized-data-importer/blob/master/tests/datavolume_test.go#L116-L125 yes they do.

Reply:
In fact it verifies both the event type and event message is what we are expecting when dealing with failures.